### PR TITLE
Entity related Records

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: pages/entity/_type/_.vue [hunk 1]
```diff
@@ -39,52 +71,126 @@
...
       };
     },
-    asyncData ({ env, params, res, redirect, app }) {
+    computed: {
+      hasResults: function () {
+        return this.searchResults.results !== null && this.searchResults.totalResults > 0;
+      },
+      entityQuery: function (params) {
+        return `http://data.europeana.eu/agent/base/${params.pathMatch}`;
+      }
```
[the remark this answers is not in the document]
Comment (lbiedinger) [reply pages/entity/_type/_.vue]: this is leftover from before, will remove.

File: components/search/SearchResultsList.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <b-list-group-item
       v-for="result in results"
       :key="result.europeanaId"
-      :to="result.linkTo"
+      :to="localePath({ name: 'record-all', params: { pathMatch: result.europeanaId.replace(/^\/+/g, '') } })"
```
Comment (rwd) [inline components/search/SearchResultsList.vue]: The `g` modifier seems redundant here given the `^` anchor.

To save using a regex, we could just do: 
```js
result.europeanaId.slice(1)
```

File: pages/entity/_type/_.vue [hunk 3]
```diff
@@ -24,11 +24,43 @@
...
+        >
+          {{ $t('noMoreResults') }}
+        </p>
+        <SearchResultsList
+          v-else
+          :results="searchResults.results"
+        />
+        <InfoMessage
+          v-if="searchResults.lastAvailablePage"
+          message="Additional results are not shown as only the first 1000 most relevant results are shown. If you haven't found what you're looking for, please consider refining your search."
```
Comment (rwd) [inline pages/entity/_type/_.vue]: This should be in the locale file.

File: pages/entity/_type/_.vue [hunk 4]
```diff
@@ -39,52 +71,123 @@
...
     data () {
       return {
         error: null,
         title: null,
-        relatedEntities: null
+        entity: null,
+        relatedEntities: null,
+        searchResults: {
+          error: null,
+          errorNoResults: 'No results',
```
Comment (rwd) [inline pages/entity/_type/_.vue]: In the locale file.

Comment (lbiedinger) [reply pages/entity/_type/_.vue]: actually it's not used here, I've removed it and moved the one in the search page(where this was copied from) into the locale file.

**Pull request merged**